libcxx/test/std/utilities/optional/optional.object/triviality.pass.cpp

```diff
@@ -95,5 +110,12 @@ int main(int, char**) {
         NonTrivialTypes::ApplyTypes<DoTestsMetafunction>{},
         DoTestsMetafunction<TrivialMoveNonTrivialCopy, TrivialCopyNonTrivialMove>{}
     );
+
+#if TEST_STD_VER >= 26
```

Comment:
Given that the properties are `static_assert`'d above, I think we can remove this part.